### ISSUE TITLE
git-duet v0.7.0 doesn't work on macOS Monterey 

### ISSUE THREAD

**Issue by pbusko** [git-duet/git-duet#99]
The latest release of git-duet (v0.7.0) was built using go version `v0.10.2` which no longer works on macOS Monterey. Rebuilding locally with the up-to-date go version fixes the issue.

```
$ git solo pb
fatal error: runtime: bsdthread_register error
runtime stack:
runtime.throw(0x11a856a, 0x21)
    /home/travis/.gimme/versions/go1.10.2.linux.amd64/src/runtime/panic.go:616 +0x81 fp=0x7ff7bfeff0c8 sp=0x7ff7bfeff0a8 pc=0x1028131
runtime.goenvs()
    /home/travis/.gimme/versions/go1.10.2.linux.amd64/src/runtime/os_darwin.go:129 +0x83 fp=0x7ff7bfeff0f8 sp=0x7ff7bfeff0c8 pc=0x1025cb3
runtime.schedinit()
    /home/travis/.gimme/versions/go1.10.2.linux.amd64/src/runtime/proc.go:501 +0xd6 fp=0x7ff7bfeff160 sp=0x7ff7bfeff0f8 pc=0x102aa26
runtime.rt0_go(0x7ff7bfeff198, 0x1, 0x7ff7bfeff198, 0x0, 0x1000000, 0x1, 0x7ff7bfeff480, 0x0, 0x7ff7bfeff498, 0x7ff7bfeff4a7, ...)
    /home/travis/.gimme/versions/go1.10.2.linux.amd64/src/runtime/asm_amd64.s:252 +0x1f4 fp=0x7ff7bfeff168 sp=0x7ff7bfeff160 pc=0x104f9d4
```

Since the `v1.16.6` is already specified in `.travis.yml` (https://github.com/git-duet/git-duet/commit/8a7deedf0f99f526010960e2833a59af569089de), it would be great to have a fresh release available, also via homebrew.

**Comment by rafecolton** [comment]
Thanks for the report. Assigning myself!

**Comment by cunnie** [comment]
Temporary workaround (assuming you have a working Golang environment, and your path includes `$HOME/go/bin`):
```zsh
brew uninstall git-duet
go install github.com/git-duet/git-duet/git-duet@latest
go install github.com/git-duet/git-duet/git-duet-commit@latest
go install github.com/git-duet/git-duet/git-solo@latest
git config --global alias.ci duet-commit
```
Disclaimer: I left out a bunch of commands (e.g. `git-duet-revert`)

**Comment by c0d1ngm0nk3y** [comment]
Any update on this? As I understood, it might "just" be as easy as creating a release with the already updated `go` version. Or am I wrong?

**Comment by rafecolton** [comment]
Thanks for the bump. To be honest, it's just not a high priority for me at the moment. I'll set myself a reminder for next week.

**Comment by jszwedko** [comment]
👍 I had a few free minutes so I started prepping the new release over in https://github.com/git-duet/git-duet/pull/100

**Comment by ravijayaraman** [comment]
@jszwedko @rafecolton Thank you taking this up, several of our engineers are currently blocked by this issue. I'm sure more people are looking for a solution soon, would appreciate if this could be fast-tracked to this week.

**Comment by tech-ganesh** [comment]
@rafecolton @jszwedko Eagerly waiting for this fix. Can you please expedite the release?

**Comment by Haegi** [comment]
I think as short-term mitigation (if it is really urgent), you can just recompile it by yourself.

**Comment by jszwedko** [comment]
Apologies for the delay, I was traveling this past week. I went to create a release today, but it failed as we previously built git-duet for darwin/386 but this is was not supported by Go 1.16. I opened #101 to fix this.

As @Haegi notes, it is possible to install manually for now as a workaround, but I'll aim to get a release out soon.

**Comment by jszwedko** [comment]
Apologies for the delay. This has been released as 0.8.0. The update to the hombrew tap is pending here: https://github.com/git-duet/homebrew-tap/pull/9
